src/operand/reference.ts

```diff
@@ -86,11 +87,44 @@ function contextValueLookup(ctx: Context, key: string): Result {
   return undefined
 }
 
+enum DataType {
+  Number = 'Number',
+  String = 'String',
+}
+
+// Equivalent to /^.+\.\((Number|String)\)$/
+const dataTypeRegex = new RegExp(
+  `^.+\\.\\((${Object.keys(DataType).join('|')})\\)$`
+)
```

Comment:
Minor (not action at this moment): I would wonder if we could convert this into a non-rx solution to save some performance.